config/environments/development.rb

```diff
@@ -1,4 +1,7 @@
 require "active_support/core_ext/integer/time"
+if Rails.env.development?
```

Comment:
This file only gets loaded in development. No need to put a conditional around it.